src/main/java/net/smartcosmos/cluster/gateway/resource/GatewayErrorController.java

```diff
@@ -88,17 +88,16 @@
                 exceptionMessage = exceptionMessage.concat(String.format(": %s", rootCause.getMessage()));
             }
 
-            String msg = String.format("Using route '%s' to service '%s' for request '%s' failed.\n"
-                                       + "Status code: '%s', Error: '%s'\n"
-                                       + "Cause: %s\n"
-                                       + "Root cause: %s",
-                                       route,
-                                       serviceId,
-                                       requestUri,
-                                       httpStatus,
-                                       errorMessage,
-                                       errorException != null ? errorException.toString() : "No exception available in context",
-                                       rootCause != null ? rootCause.toString() : "N/A");
+            String msg = new StringBuilder() // @formatter:off
```

Comment:
I agree with @andreassiegelrfid, I don't think this makes it more readable.  I prefer the `String.format()`.